tests/unit/test_engine.py

```diff
@@ -316,26 +316,26 @@ class Model(BaseModel):
 
 
 def test_load_dump_subclass(engine):
-    """Only the immediate Columns of a model should be dumped"""
+    """Should handle subclasses"""
 
     class Admin(User):
-        admin_id = Column(Integer, hash_key=True)
+        admin_id = Column(Integer)
         other = Column(Integer)
-    engine.bind(User)
+    engine.bind(Admin)
 
     admin = Admin(admin_id=3)
     # Set an attribute that would be a column on the parent class, but should
     # have no meaning for the subclass
     admin.email = "admin@domain.com"
 
-    dumped_admin = {"admin_id": {"N": "3"}}
+    dumped_admin = {'admin_id': {'N': '3'}, 'email': {'S': 'admin@domain.com'}}
     assert engine._dump(Admin, admin) == dumped_admin
 
     # Inject a value that would have meaning for a column on the parent class,
```

Comment:
This comment is out of date, the value should have meaning for the subclass as well.